src/components/ui/copy-to-clipboard-button.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import { CheckIcon } from "@/components/ui/check-icon";
+import { ClipboardIcon } from "@/components/ui/clipboard-icon";
+import { cn } from "@/lib/utils";
+
+export function CopyToClipboardButton({
+	generatedAnswer,
+}: {
+	generatedAnswer: string;
+}) {
+	const [isCopiedToClipboard, setIsCopiedToClipboard] = useState(false);
+
+	const copyToClipboard = async () => {
+		await navigator.clipboard.writeText(generatedAnswer);
+		setIsCopiedToClipboard(true);
+	};
+
+	/**
+	 * reset icon after 2 seconds
+	 */
+	useEffect(() => {
+		const timeoutId = setTimeout(() => setIsCopiedToClipboard(false), 2000);
+
+		return () => clearTimeout(timeoutId);
+	}, [isCopiedToClipboard]);
+
+	return (
+		<>
+			<button
+				data-testid="copy-answer-to-clipboard"
+				className={`group text-parla-icon-grey ${isCopiedToClipboard ? "hover:text-parla-icon-black" : "hover:text-parla-icon-blue"}`}
+				onClick={copyToClipboard}
+			>
+				{isCopiedToClipboard ? <CheckIcon /> : <ClipboardIcon />}
+				<div className="group-hover:block relative hidden ">
+					<div
+						className={cn(
+							"absolute",
+							"top-1.5",
+							"left-1/2",
+							"-translate-x-1/2",
+							"w-[5.5rem]",
+							"z-20",
+							"bg-parla-icon-grey",
+							"text-white",
+							"text-center",
+							"rounded-xl",
+							"text-sm",
+							"p-2",
+							// the after: classes represent the little tooltip tail
+							'after:contents-[""]',
+							"after:z-10",
+							"after:absolute",
+							"after:-top-1",
+							"after:left-1/2",
+							"after:-ml-1",
+							"after:rotate-45",
+							"after:border-4",
+							"after:border-solid",
+							"after:border-parla-icon-grey",
+						)}
+					>
+						{isCopiedToClipboard ? "Kopiert!" : "Kopieren"}
```

Comment:
Kopiert and Kopieren should be added to texts.tsx

Reply:
yes, my bad 👍